### DISCUSSION TITLE
WooCommerce mini cart hooks

### DISCUSSION THREAD

**Discussion by YossiDavid** [elementor/elementor#25144] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

The feature I ask for will help developers and non-coders to improve the possibilities of the mini cart template by adding to it all the hooks of woo mini cart.

### Describe the solution you'd like.

Adding to woo mini cart template the following hooks like the image: https://unlimitedwp.com/wp-content/uploads/2023/08/woocommerce-mini-cart-hooks.png

Currently, there is just the first two hook:
1. woocommerce_before_mini_cart
2. woocommerce_mini_cart_content

I would like to add the following also:
1. woocommerce_widget_shopping_cart_total
2. widget_shopping_cart_before_button
3. widget_shopping_cart_after_button
4. woocommerce_after_mini_cart

### Describe alternatives you've considered.

For now the only alternative is to use my own mini cart template in the child theme, but I prefer of course to use Elementor's one.

### Additional context

https://unlimitedwp.com/wp-content/uploads/2023/08/woocommerce-mini-cart-hooks.png

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by tiago-sartor** [comment]
Are you still able to use 'woocommerce_before_mini_cart' and 'woocommerce_mini_cart_content' inside Elementor's mini cart widget? They don't seem to be working for me...
